src/addons/cells/headerCells/NumberFilterableHeaderCell.js

```diff
@@ -0,0 +1,86 @@
+const React              = require('react');
+const ExcelColumn        = require('../../grids/ExcelColumn');
+
+const NumberFilterableHeaderCell = React.createClass({
+
+  propTypes: {
+    onChange: React.PropTypes.func.isRequired,
+    column: React.PropTypes.shape(ExcelColumn)
+  },
+
+  getInitialState(): {filterTerm: string} {
+    return {filterTerm: '', filterArray:[]};
+  },                                                 
+      
+  handleChange(e: Event) {
+    let val = e.target.value;
+    var tmpStrArray = val.split(",");        
+    var tmpCurrentState = []; 
+    var filterArray = [];
+   
+    if(isNaN(tmpStrArray)){
+    for (var tmpStrArrayVal in tmpStrArray) {        
+        if(tmpStrArray[tmpStrArrayVal]){       
+            var contentString = tmpStrArray[tmpStrArrayVal];
+            if(contentString.indexOf("-")!=-1){               
+                var strSplit = contentString.split("-");
+                var firstNumber = parseInt(strSplit[0]);
```

Comment:
```let minRange = parseInt(strSplit[0], 10);```
You need to pass a radix value of 10 as a second parameter so that it parses the input as a decimal